static/src/javascripts/projects/common/modules/commercial/acquisitions-copy.js

```diff
@@ -32,96 +39,81 @@ const ctaLinkSentenceWorldCupLiveblogPlayful = (
 ): string =>
     `<span class="contributions__highlight"> For as little as ${currencySymbol}1, you can support the Guardian – and it only takes a minute.</span> Otherwise we’ll send Sergio Ramos round to sort you out. <a href="${supportUrl}" target="_blank" class="u-underline">Make a contribution</a>`;
 
-const thankyouP2FirstSentence =
-    'Thank you to the many people who have already supported us financially &ndash; your contribution is what makes stories like you’ve just read possible. We increasingly need our readers to fund our work so that we can continue holding power to account and producing fearless journalism.</br></br>';
+export const getCopyFromGoogleDoc = (
+    sheetName: string
+): Promise<AcquisitionsEpicTemplateCopy> => {
+    const url =
+        'https://interactive.guim.co.uk/docsdata/1fy0JolB1bf1IEFLHGHfUYWx-niad7vR9K954OpTOvjE.json';
+
+    return fetchJSON(url, {
+        mode: 'cors',
+    }).then(res => {
```

Comment:
Is there a way we could cache this response? Currently, it makes the call on every variant of every test

Reply:
(probably not for this PR)

Reply:
yeah that's a nice idea! we do at least have Fastly in front of `https://interactive.guim.co.uk` so responses should be quick (it's just some data in s3). But certainly making a request per sheet is inefficient and easily fixed. As for caching responses between page loads, that might require a bit more thought but we could put it in localStorage

Reply:
OK, so I've managed to solve both this and the problem below where we're trying to get data for variants that aren't running.

The issue was that requests are made when the test is defined, not inside the test function. So I made `makeABTestVariant` accept a function as `option.copy`, also also made the control a function. Now, the function gets executed inside the test function. So it will only run for one variant which is actually active on the page.

Reply:
Great!